src/main/python/net/i2cat/cnsmo/app/vpn/configurator.py

```diff
@@ -121,22 +121,22 @@ def generate_ca_and_dh(self):
         command = "sh %s../gen_ca.sh" % self.key_dir
         subprocess.check_call(shlex.split(command), shell=False)
 
-    def generate_client_certs(self):
-        command = "sh %s../gen_client.sh" % self.key_dir
+    def generate_client_certs(self, client_id):
+        command = "sh %s../gen_client.sh -u %s" % (self.key_dir, client_id)
```

Comment:
Passing an unchecked user given parameter (client_id) to the script is a source of injection attacks